docs/sections/user_guide/cli/drivers/cdeps/index.rst

```diff
@@ -1,23 +1,11 @@
 ``cdeps``
 =========
 
-.. include:: /shared/idempotent.rst
+.. include:: ../shared/idempotent.rst
 
 The ``uw`` mode for configuring and running the :CDEPS:`cdeps<>` component.
 
-.. literalinclude:: cdeps/help.cmd
-   :language: text
-   :emphasize-lines: 1
-.. literalinclude:: cdeps/help.out
-   :language: text
-
-All tasks take the same arguments. For example:
-
-.. literalinclude:: cdeps/run-help.cmd
-   :language: text
-   :emphasize-lines: 1
-.. literalinclude:: cdeps/run-help.out
-   :language: text
```

Comment:
Here and just below are two repetitive blocks that are now factored out of each driver `index.rst` file and replaced with an `include` directive. The diffs in this file and its directory repeat many times below for each of the other drivers.